### ISSUE TITLE
strange interaction with defaults for speed factor distribution

### ISSUE THREAD

**Issue by behrisch** [eclipse-sumo/sumo#15173]
The default distribution for speedFactor is normc(1.0, 0.1, 0.2, 2.0). This has the side effect that you cannot always set speedFactor=3 because this violates the bounds: It is only possible if you at the same time define speedDev=0 (because setting this disables bound checking) or do a complete distribution description.

**Comment by namdre** [comment]
maybe adapt the upper bound to  MAX(defaultUbberBound, `mean + 2 * speedDev`) ? when only setting speedFactor to a number?

**Comment by behrisch** [comment]
Probably we would need something for the lower bound as well but yes this sounds like an option although to be consistent with the default we would need to do something like `MAX(defaultUpperBound, 2 * mean)` or `MAX(defaultUpperBound, mean + 10 * speedDev)`?

**Comment by namdre** [comment]
If `defaultUpperBound` is defined a *2* than we can put anything we want in the second term of the MAX or am I overlooking something?

**Comment by behrisch** [comment]
No, but your proposal means that setting a speedfactor of 3 gives normc(3.0, 0.1, 0.2, 3.2) which gives in some sense less "headspace" than normc(1.0, 0.1, 0.2, 2.0) because the cutoff already happens at two standard deviations and not at 10 as in the default. I am fine with that just wanted to mention that it might be not the expectation of the user.

**Comment by namdre** [comment]
I think the large default max of 2 was chosen in part to compensate for the lack of automatic max adjustment and not for the explicit desired to keep 10 * stddev.

**Comment by behrisch** [comment]
OK but then we should adapt the bounds also if the user only sets a speed deviation. So basically if the user does not define the bounds herself, we use `MAX(defaultUpperBound, mean + 2 * speedDev)` and `MIN(defaultLowerBound, mean - 2 * speedDev)` ?
